aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -624,6 +575,83 @@ HTTP/2 is enabled by default. Connections fall back to HTTP/1.1 if an HTTP/2 con
+* App's hosted [in-process](xref:fundamentals/servers/index#in-process-hosting-model) can be configured not to timeout during periods of inactivity.
+
+### Application Initialization Module
+
+*Applies to apps hosted in-process and out-of-process.*
+
+[IIS Application Initialization](/iis/get-started/whats-new-in-iis-8/iis-80-application-initialization) is an IIS feature that sends an HTTP request to the app when the app pool starts or is recycled. The request triggers the app to start. By default, IIS issues a request to the app's root URL (`/`) to initialize the app (see the [additional resources](#application-initialization-module-and-idle-timeout-additional-resources) for details on further configuration).
+
+Confirm that the IIS Application Initialization role feature in enabled:
+
+On Windows 7 or later desktop systems when using IIS locally:
+
+1. Navigate to **Control Panel** > **Programs** > **Programs and Features** > **Turn Windows features on or off** (left side of the screen).
+1. Open **Internet Information Services** > **World Wide Web Services** > **Application Development Features**.
+1. Select the check box for **Application Initialization**.
+
+On Windows Server 2008 R2 or later:
+
+1. Open the **Add Roles and Features Wizard**.
+1. When you reach the **Select role services** panel, open the **Application Development** node.
+1. Select the check box for **Application Initialization**.
+
+Use either of the following approaches to enable the Application Initialization Module for the site:
+
+* Using IIS Manager:
+
+  1. Select **Application Pools** in the **Connections** panel.
+  1. Right-click the app's app pool in the list and select **Advanced Settings**.
+  1. Set **Start Mode** to **AlwaysRunning**. Select **OK**.
+  1. Open the **Sites** node in the **Connections** panel.
+  1. Right-click the app and select **Manage Website** > **Advanced Settings**.
+  1. Set **Preload Enabled** to **True**.
+
+* Using *web.config*, add the `<applicationInitialization>` element with `doAppInitAfterRestart` set to `true` to the `<system.webServer>` elements in the app's *web.config* file:
+
+  ```xml
+  <?xml version="1.0" encoding="utf-8"?>
+  <configuration>
+    <location path="." inheritInChildApplications="false">
+      <system.webServer>
+        <applicationInitialization doAppInitAfterRestart="true" />
+      </system.webServer>
+    </location>
+  </configuration>
+  ```
+
+### Idle Timeout
+
+*Only applies to apps hosted in-process.*
+
+To prevent the app from idling, set the app pool's idle timeout using IIS Manager:
+
+1. Select **Application Pools** in the **Connections** panel.
+1. Right-click the app's app pool in the list and select **Advanced Settings**.
+1. Set the **Idle Time-out (minutes)** to **0** (zero). Select **OK**.
```

Comment:
What is the default for the idle timeout?

Reply:
20 minutes ... I'll make a note of it on the next commit.

... and I don't currently say that `OnDemand` is the default **Start Mode**. I may as well add that, too.